File: nas_spec/code_samples/C#/metadata@card/post.cs
// For more information please refer to https://github.com/checkout/checkout-sdk-net
using Checkout.Metadata.Card;
using Checkout.Metadata.Card.Source;

//API keys
ICheckoutApi api = CheckoutSdk.Builder().StaticKeys()
    .SecretKey("secret_key")
    .Environment(Environment.Sandbox)
    .HttpClientFactory(new DefaultHttpClientFactory())
    .Build();

//OAuth
ICheckoutApi api = CheckoutSdk.Builder().OAuth()
    .ClientCredentials("client_id", "client_secret")
    .Scopes(OAuthScope.VaultCardMetadata)
    .Environment(Environment.Sandbox)
    .HttpClientFactory(new DefaultHttpClientFactory())
    .Build();

CardMetadataRequest request = new CardMetadataRequest
{
    Source = new CardMetadataCardSource { Number = "4242424242424242" },
    Format = CardMetadataFormatType.Basic
};

try
{
    CardMetadataResponse response = await api.MetadataClient().RequestCardMetadata(request);
}
catch (CheckoutApiException e)
{
    // API error
    string requestId = e.RequestId;
    var statusCode = e.HttpStatusCode;
    IDictionary<string, object> errorDetails = e.ErrorDetails;
}
catch (CheckoutArgumentException e)
{
    // Bad arguments
}
catch (CheckoutAuthorizationException e)
{
    // Invalid authorization
}

File: nas_spec/code_samples/PHP/metadata@card/post.php
<?php
//For more information please refer to https://github.com/checkout/checkout-sdk-php
use Checkout\\CheckoutApiException;
use Checkout\\CheckoutAuthorizationException;
use Checkout\\CheckoutSdk;
use Checkout\\Environment;
use Checkout\\OAuthScope;
use Checkout\\Metadata\\Card\\CardMetadataFormatType;
use Checkout\\Metadata\\Card\\CardMetadataRequest;
use Checkout\\Metadata\\Card\\Source\\CardMetadataCardSource;

//API Keys
$api = CheckoutSdk::builder()->staticKeys()
    ->environment(Environment::sandbox())
    ->secretKey("secret_key")
    ->build();

$api = CheckoutSdk::builder()->oAuth()
    ->clientCredentials("client_id", "client_secret")
    ->scopes([OAuthScope::$VaultCardMetadata])
    ->environment(Environment::sandbox())
    ->build();

$cardMetadataRequest = new CardMetadataRequest();
$cardMetadataRequest->format = CardMetadataFormatType::$BASIC;
$cardMetadataRequest->source = new CardMetadataCardSource();
$cardMetadataRequest->source->number = "4242424242424242";

try {
    $response = $api->getMetadataClient()->requestCardMetadata($cardMetadataRequest);
} catch (CheckoutApiException $e) {
    // API error
    $error_details = $e->error_details;
    $http_status_code = isset($e->http_metadata) ? $e->http_metadata->getStatusCode() : null;
} catch (CheckoutAuthorizationException $e) {
    // Bad Invalid authorization
}